xbmc/utils/LangCodeExpander.h

```diff
@@ -140,6 +141,22 @@ class CLangCodeExpander
   std::vector<std::string> GetLanguageNames(LANGFORMATS format = ISO_639_1,
                                             LANG_LIST list = LANG_LIST::DEFAULT);
 
+  /** \brief Performs various ISO 3166-1 lookups.
+  *          Input can be alpha-2, alpha-3 or name.
+  *          If the input is alpha-2 or alpha-3 then the
+  *          name will be returned unless returnCode is true,
+  *          then the opposite code will be returned.
+  *          If the input is a name, then alpha-3 is returned
+  *          unless returnCode is true, then alpha-2 is returned.
+  *   \param[in] searchString The code or name to be found.
+  *   \param[out] returned The code or name being returned.
+  *   \param[in] returnCode Control the value returned.
+  *   \return true if the conversion succeeded, false otherwise.
+  */
+  bool LookupISO31661(std::string_view searchString,
+                      std::string& returned,
+                      bool returnCode = false);
```

Comment:
incorrect description format
```suggestion
  /*!
   * \brief Performs various ISO 3166-1 lookups.
   *          Input can be alpha-2, alpha-3 or name.
   *          If the input is alpha-2 or alpha-3 then the
   *          name will be returned unless returnCode is true,
   *          then the opposite code will be returned.
   *          If the input is a name, then alpha-3 is returned
   *          unless returnCode is true, then alpha-2 is returned.
   *   \param[in] searchString The code or name to be found.
   *   \param[out] returned The code or name being returned.
   *   \param[in] returnCode Control the value returned.
   *   \return true if the conversion succeeded, false otherwise.
   */
  bool LookupISO31661(std::string_view searchString,
                      std::string& returned,
                      bool returnCode = false);
```
And variable names can be improved
searchString --> could be searchCode
returned --> convCode 
returnCode --> this is unclear and also the description is unclear:
`\param[in] returnCode Control the value returned.`
variable name and description must be changed in a more clear way

Reply:
Description format updated and variable names altered as suggested.